packages/rx-utils/src/rd/entity-store.utils.ts

```diff
@@ -127,6 +191,17 @@ export class EntityStore<L = never, A = never> {
 		);
 	}
 
+	/**
+	 * Update an entity
+	 *
+	 * @param key
+	 * key for an entity you want to update.
+	 *
+	 * @param update
+	 * Describes how to update an entity.
+	 * Returns a stream with updated entity.
```

Comment:
jsdoc has @returns as far as I remember. Let's make all the doc comments uniform regarding return value describing.

Reply:
Actually, I thought about it, but I'm not sure if we really need it:
1) If we want to add `@returns` param, so why not to add type annotations for other params?
2) But we already have a TypeScript here and all types will be available in a d.ts file.

And for me, it looks like breaking `Don't Repeat Yourself` principle :)

Soo, what do you think about it: should I add only `@return` param, or also add types for other params or maybe it's better just to leave it as it is?

Reply:
jsdoc is very convenient for generating docs, and this is quite actual for us :) Much harder to find a tool which would use both jsdoc and TS type annotations for doc generation. I agree that we will have to keep them consistent and that's additional work, but I'm still for this.

Reply:
We should never use jsdoc for type annotations but only for descriptions and meta information (additional tags etc.). If `@returns` supports description then we should add it showing what exactly does the method return. Otherwise skip it.